apps/web/src/pages/Pools/AssessmentPlanBuilderPage/components/AssessmentDetailsDialogLabels.ts

```diff
@@ -1,40 +1,42 @@
+import { defineMessage } from "react-intl";
+
 const labels = {
-  typeOfAssessment: {
+  typeOfAssessment: defineMessage({
```

Comment:
Why not use `defineMessages()` and add them all to one list instead?

Reply:
Great question @yonikid15. I was looking at https://github.com/GCTC-NTGC/gc-digital-talent/pull/9387/files#diff-6a8b4febf77528846719a188df43825d0ec10d309c810928f0a816d35d1245a6 and followed that pattern. The codebase doesn't stick to one way of doing it, but we probably should use `defineMessages` when possible.

Updated: 156b46e620ada58e4f6ae1080efa5531094467f7.

Reply:
Ah I see, I thought it was to preserve the id's 😆. Ya we should pick a pattern and stick it. If there's more than 2 items then use `defineMessages`.